commands/comment.go

```diff
@@ -43,7 +43,13 @@ var (
 )
 
 func init() {
-	commentFlagSet.Var(&commentLocation, "l", "File location be commented upon; requires that the -f flag also be set")
+	commentFlagSet.Var(&commentLocation, "l",
+		`File location be commented upon; requires that the -f flag also be set.
+Location follows the following format:
+    <START LINE>[+<START COLUMN>][:<END LINE>[+<END COLUMN>]]
+So in order to comment starting on the 5th character of the 2nd line until (and
```

Comment:
I think there should be a comma between "So" and "in".

i.e. "So, in order ..." instead of "So in order ..."

Reply:
Thanks for catching that! My English is getting a little bit more rusty every day, but correct me if I am wrong that I should (or could?) put a comma before the "use:":
"the 7th line use:" => "the 7th line, use:"

Reply:
@llorens I *believe* the comma makes sense there, but my grammar knowledge is very primitive.